packages/fuels-test-helpers/Cargo.toml

```
@@ -9,9 +9,9 @@ repository = "https://github.com/FuelLabs/fuels-rs"
 description = "Fuel Rust SDK test helpers."
 
 [dependencies]
-fuel-core = { version = "0.8", default-features = false, optional = true }
+fuel-core = { version = "0.9", default-features = false, optional = true }
 fuel-core-interfaces = { version = "0.8", default-features = false }
```

This should be bumped as well I think?
```suggestion
fuel-core-interfaces = { version = "0.9", default-features = false }
```